Bulletin & version update for 0.4.4 (#4937)
Significant-Gravitas/Auto-GPT: BULLETIN.md

Reviewer: Whitespace between headings and text isn't consistent

Reviewer: To make sure it's all pretty in the terminal, lines should be truncated to 88 characters or less (just like the rest of the codebase). (This could be solved in code as well, in the function that converts markdown to terminal text.)

# QUICK LINKS 🔗
# --------------
🌎 *Official Website*: https://agpt.co.
📖 *User Guide*: https://docs.agpt.co.
👩 *Contributors Wiki*: https://github.com/Significant-Gravitas/Auto-GPT/wiki/Contributing.

# v0.4.4 RELEASE HIGHLIGHTS! 🚀

> # ------------------------------

Reviewer: ```suggestion
# -----------------------------
```

## GPT-4 is back!

> Following OpenAI's recent GPT-4 GA announcement, the SMART_LLM .env setting 

Reviewer: GA is not a common acronym afaik

The documentation also has a place here, at *https://docs.agpt.co*

# For contributors 👷🏼
Check out the contribution guide on our wiki:
https://github.com/Significant-Gravitas/Auto-GPT/wiki/Contributing

# 🚀 v0.4.4 Release Highlights! 🚀
### GPT-4 is back!

> Following OpenAI's GPT-4 GA announcement, SMART_LLM now defaults to GPT-4.

Reviewer: ```suggestion
Following OpenAI's GPT-4 General Availability announcement, Auto-GPT will use GPT-4
by default in its main run-loop.
```

Author: Updated, but kept the reference to SMART_LLM as the users may want to know how to change this.

> **⚠️💀$$ HIGH COSTS WARNING \$\$ 💀⚠️** 
> - GPT-4 costs ~20x more than GPT-3.5-turbo. Please be aware of this when using SMART_LLM.
> - You can use `--gpt3only` or `--gpt4only` to force the use of GPT-3.5-turbo or GPT-4, respectively, at runtime

### Re-arch v1 preview released!

> We're released a preview version of the re-arch code, under `autogpt/core`. This is a major milestone for us, and we're excited to continue working on it. We look forward to your feedback. Follow the process [here]([Title](https://github.com/Significant-Gravitas/Auto-GPT/issues/4770)).

Reviewer: It may be markdown, but link conversion doesn't work for the terminal display. :)
```suggestion
We're released a preview version of the re-arch code, under `autogpt/core`. This is a major milestone for us, and we're excited to continue working on it. We look forward to your feedback. Follow the process here: https://github.com/Significant-Gravitas/Auto-GPT/issues/4770
```

Author: Fixed.
